clients/src/main/java/org/apache/kafka/common/protocol/SecurityProtocol.java

```diff
@@ -26,6 +26,8 @@
     PLAINTEXT(0, "PLAINTEXT"),
     /** SSL channel */
     SSL(1, "SSL"),
+    /** SASL authenticated, non-encrypted channel */
+    PLAINTEXTSASL(2, "PLAINTEXTSASL"),
```

Comment:
@ijuma  I do prefer SASL_PLAINTEXT and SASL_SSL since it is clearer (more readable) than SSLSASL and PLAINTEXTSASL.